metainfo.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+  The following file from the 2.6 branch titan service was used as a template:
+  https://github.com/apache/ambari/blob/branch-2.6/ambari-server/src/main/resources/common-services/TITAN/1.0.0/metainfo.xml
+-->
+<metainfo>
+  <schemaVersion>2.0</schemaVersion>
+  <services>
+    <service>
+      <name>JANUSGRAPH</name>
+      <displayName>JanusGraph</displayName>
+      <comment>JanusGraph is a scalable graph database optimized for storing and querying graphs containing hundreds of
+        billions of vertices and edges distributed across a multi-machine cluster.</comment>
+      <version>0.3.1</version>
+      <components>
+        <component>
+          <name>JANUSGRAPH_SERVER</name>
+          <displayName>JanusGraph Server</displayName>
+          <category>MASTER</category>
+          <cardinality>1</cardinality>
+          <commandScript>
+            <script>scripts/janusgraph_server.py</script>
+            <scriptType>PYTHON</scriptType>
+            <timeout>600</timeout>
+          </commandScript>
+          <configFiles>
+            <configFile>
+              <type>yaml</type>
+              <fileName>gremlin-server.yaml</fileName>
+              <dictionaryName>gremlin-server</dictionaryName>
+            </configFile>
+          </configFiles>
+        </component>
+<!--        <component>
```

Comment:
If you're not adding JanusGraph client support with this PR, remove this commented code. You could open an issue for client support and add this snippet in there. I'll also note that for client support, you'd need to update the `conf/remote.yaml` and `conf/remote-objects.yaml` from `localhost` to the hostname of the JanusGraph server.